tests/conftest.py

```diff
@@ -1026,19 +1069,315 @@ def get_state_db(self):
 
         return self.state_db
 
+class DockerVirtualChassisTopology(object):
+    def __init__(self, namespace=None, chassbr=None, imgname=None, keeptb=False,
+                 fakeplatform=None, topoFile=None):
+        self.ns = namespace
+        self.chassbr = "br4chs"
+        self.keeptb = keeptb
+        self.fakeplatform = fakeplatform
+        self.topoFile = topoFile
+        self.imgname = imgname
+        self.ctninfo = {}
+        self.dvss = {}
+        self.inbands = {}
+        if self.ns is None:
+           self.ns = random_string()
+        print("VCT ns: " + self.ns)
+        self.find_all_ctns()
+        with open(self.topoFile, "r") as f:
+            self.virt_topo = json.load(f)["VIRTUAL_TOPOLOGY"]
+        self.oper = "create"
+        self.handle_request()
+
+    def runcmd( self, cmd, addns=True ):
+        try:
+            netns = ""
+            if addns:
+               netns = "sudo ip netns exec %s" % self.ns
+            subprocess.check_output("%s %s" % (netns, cmd),
+                                    stderr=subprocess.STDOUT, shell=True)
+        except subprocess.CalledProcessError as e:
+            print "------rc={} for cmd: {}------".format(e.returncode, e.cmd)
+            print e.output.rstrip()
+            print "------"
+            return e.returncode
```

Comment:
I think we have switched to python3, it will like to fail for now.

Reply:
ack. the build failed this time..but not the previous builds